linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/grpc/GrpcClassifier.scala

```diff
@@ -10,9 +10,12 @@ import io.buoyant.grpc.runtime.GrpcStatus.{Ok, Unavailable}
 
 trait GrpcClassifier extends H2Classifier {
 
+  val successStatusCodes: Set[Int]
```

Comment:
for the sake of symmetry, it would be nice to not define this val and just to leave the success def abstract, just like we do for retryable.